### PR TITLE
Kddimitrov/webpack source maps log

### PR DESCRIPTION
**Pull request by KristianDD**
## PR Checklist

- [ ] The PR title follows our guidelines: https://github.com/NativeScript/NativeScript/blob/master/CONTRIBUTING.md#commit-messages.
- [ ] There is an issue for the bug/feature this PR is for. To avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it.
- [ ] You have signed the [CLA](http://www.nativescript.org/cla).
- [ ] All existing tests are passing: https://github.com/NativeScript/nativescript-cli/blob/master/CONTRIBUTING.md#contribute-to-the-code-base
- [ ] Tests for the changes are included.

## What is the new behavior?

Fixes/Implements/Closes #[Issue Number].

### CODE REVIEW

File: test/services/log-source-map-service.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+	return testInjector;
+}
+
+function toPlatformSep(filePath: string) {
+	return stringReplaceAll(filePath, "/", path.sep);
+}
+
+const testCases: IDictionary<Array<{caseName: string, message: string, expected: string}>> = {
+	"android": [{
+			caseName: "trace massage",
```
Comment (rosen-vladimirov) [inline test/services/log-source-map-service.ts]: not a merge stopper, but I assume you meant message not massage (or it is just Thursday 😸 )

File: test/services/log-source-map-service.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+			expected: `CONSOLE LOG Test. file:///${toPlatformSep("src/main-view-model.ts")}:29:20\n`
+		}, {
+			caseName: "trace massage",
+			message: "CONSOLE TRACE file:///app/bundle.js:295:22: Test",
+			expected: `CONSOLE TRACE Test file:///${toPlatformSep("src/main-view-model.ts")}:30:22\n`
+		}, {
+			caseName: "error massage",
+			message: "file:///app/bundle.js:296:32: JS ERROR Error: Test",
+			expected: `JS ERROR Error Test file:///${toPlatformSep("src/main-view-model.ts")}:31:31\n`
+		}]
```
Comment (rosen-vladimirov) [inline test/services/log-source-map-service.ts]: Can you please add test in which the original file location cannot be found and ensure we output the original line in this case?

File: lib/common/mobile/ios/device/ios-application-manager.ts [hunk 3]
```diff
@@ -95,6 +95,7 @@ export class IOSApplicationManager extends ApplicationManagerBase {
 
 	private async setDeviceLogData(appData: Mobile.IApplicationData): Promise<void> {
 		this.$deviceLogProvider.setProjectNameForDevice(this.device.deviceInfo.identifier, appData.projectName);
+		this.$deviceLogProvider.setProjectDirForDevice(this.device.deviceInfo.identifier, appData.projectDir);
```
[the remark this answers is not in the document]
Comment (KristianDD) [reply lib/common/mobile/ios/device/ios-application-manager.ts]: I don't think it is necessary as we will overwrite it every time we start a different application. Until then the data will be valid. This is one of the differences with android devices, where the pid changes with every restart and we use it in the filter.

**Comment by KristianDD** [comment]
run ci

**Pull request merged**